opencog/persist/zmq/atomspace/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ ADD_CUSTOM_COMMAND(
 )
 
 ADD_LIBRARY (zmqatoms SHARED
-	ZMQMessages.pb
+	ZMQMessages.pb.cc ZMQMessages.pb.h
```

Comment:
strange, it builds for me ... i get the feeling you are using a different version of cmake..